index.js

```diff
@@ -145,11 +152,11 @@ export default class StarReview extends Component {
   star(val,starImg){
     return(
       <View key={val} style={{paddingLeft: this.props.spacing/2, paddingRight: this.props.spacing/2}}>
-        <TouchableOpacity onPress={()=>{
+        <TouchableOpacity disabled={this.props.disabled} onPress={()=>{
           this.setState({rating: val});
           this.props.update(val);
         }}>
-          <Image style={{width: this.props.starSize, height: this.props.starSize}} source={starImg} />
+          <Image style={[{width: this.props.starSize, height: this.props.starSize}, {tintColor: this.props.tintColor}]} source={starImg} />
```

Comment:
Is there reason why this needs style array syntax? I'm not too familiar with its use but I'm wondering what it adds here.